Src/java/engine/src/main/java/org/opencds/cqf/cql/engine/execution/Context.java

```diff
@@ -617,7 +617,7 @@ private FunctionDef getResolvedFunctionDef(final String libraryName, final Strin
         }
 
         if (candidateDefs.size() > 1 && !hasSignature) {
-            throw new CqlException(String.format("Signature not provided for overloaded function '%s'", mangledFunctionName));
+            logger.debug("Using runtime function resolution for '{}'. It's recommended to always signatures in ELM", mangledFunctionName);
```

Comment:
Error message seems like it's missing some words.  "always provide signatures in ELM"?

Reply:
Whoops. Thanks.